src/views/single-event.php

```diff
@@ -18,7 +18,21 @@
 $events_label_singular = tribe_get_event_label_singular();
 $events_label_plural   = tribe_get_event_label_plural();
 
-$event_id = get_the_ID();
+/**
+ * Supplies the event ID and if none is specified the current post is used.
+ * 
+ * @since TBD
+ */
+$event_id = Tribe__Events__Main::postIdHelper( get_the_ID() );
+
+/**
+ * Allows filtering the event ID while searching for it.
+ * 
+ * @since TBD
+ * 
+ * @param int get_the_ID(
+ */
+$event_id = apply_filters( 'tec_events_single_event_id', get_the_ID() );
```

Comment:
Make sure the docblock above makes sense.

```suggestion
$event_id = apply_filters( 'tec_events_single_event_id', $event_id );
```